src/nrnmpi/mpispike.cpp

```diff
@@ -643,11 +643,10 @@ double nrnmpi_dbl_allreduce(double x, int type) {
 }
 
 extern "C" void nrnmpi_dbl_allreduce_vec(double* src, double* dest, int cnt, int type) {
-    int i;
     MPI_Op t;
     assert(src != dest);
     if (nrnmpi_numprocs < 2) {
-        for (i = 0; i < cnt; ++i) {
```

Comment:
Should we delete this shortcut?

Reply:
I guess there is no good reason for it to exist. There are 8 of these in the file. I guess it is just a matter of energy.